docs/server/guides/setup/minecraft-setup.md

```diff
@@ -178,6 +179,36 @@ tmux a -t sessionName
 
 To destroy the session completely, you can attach to the session and then ++ctrl+d++ out of it. It detaches from session and destroys it too.
 
+#### Setup With Screen
+
+Start a screen session
+
+```bash
+screen -S sessionName
+```
+
+`sessionName` being whatever you want to name the instance, for example `minecraft-session`
+
+This will automatically take you inside the screen session. Start the server using
+
+```bash
+java -Xmx1024M -Xms1024M -jar server.jar nogui
```

Comment:
Here you should reference to the startup script which was created a little bit further up (line 104)